ts/sagas/startup.ts

```diff
@@ -461,8 +462,9 @@ export function* initializeApplicationSaga(
   yield* call(trackKeychainGetFailure, keychainError);
   yield* call(clearKeychainError);
 
+  yield* call(checkConfiguredPinSaga);
```

Comment:
Why did you move this here? Is there an edge cases or an unknown bug?

Reply:
checking for pin existence was previously performed only when a previous session (or pin) was missing. We need to check the pin even if there is a previous session (or pin). By moving this check outside of that condition, we can perform it every time and consequently we can verify if the pin respects the rules (if fast login is enabled)